crates/bevy_core/src/bytes.rs

```diff
@@ -1,7 +1,14 @@
-use bevy_math::{Mat4, Vec2, Vec3, Vec4};
-
 pub use bevy_derive::Bytes;
 
+// NOTE: we can reexport common traits and methods from bytemuck to avoid requiring dependency most of
+// the time, but unfortunately we can't use derive macros that way due to hardcoded path in generated code.
+pub use bytemuck::{bytes_of, cast_slice, Pod, Zeroable};
+
+// FIXME: `Bytes` trait doesn't specify the expected encoding format,
+// which means types that implement it have to know what format is expected
+// and can only implement one encoding at a time.
+// TODO: Remove `Bytes` and `FromBytes` in favour of `crevice` crate.
```

Comment:
If we could make an issue for this (once this PR gets merged) that way we we're able to track the progress for this TODO.

Reply:
Sure, but It's not really part of this PR. There is some progress in #1931 and afaik cart has some private branch going on as well.